QUEST.tsv

```diff
@@ -122,82 +122,82 @@ xQUEST_20150317_000121	This mission makes me feel like I've seen enough monsters
 xQUEST_20150317_000136	Machine Worker
-xQUEST_20150317_000137	Even with the world as it is, we can't stop working. {nl}It's been four years. Isn't it time to make peace and start over?
+xQUEST_20150317_000137	Even with the world as it is, we can't stop working. {nl}It's been four years. Isn't it time to make peace with it and start over?
 xQUEST_20150317_000138	Klaipeda is a town that lives off its mining. {nl}With the Day of Divine Tree, we'd ought to work on placing trees or cutting them. 
-xQUEST_20150317_000139	My old man did this thing where he checked around corners for anything good. {nl}Sometimes, he'd get lucky and find a nice treasure chest. I wonder who leaves it there?
+xQUEST_20150317_000139	My old man did this thing where he checked around corners for anything nice. {nl}Sometimes, he'd get lucky and find a nice treasure chest. I wonder who leaves it there?
 xQUEST_20150317_000140	Retired Hunter
-xQUEST_20150317_000141	I remember a great battle in a deep canyon, our soldiers fought a huge monster. {nl}It was amazing how the monster was turned around, and there was some great spoils, too.
+xQUEST_20150317_000141	I remember this battle in a deep canyon, where soldiers fought a huge monster. {nl}It was amazing how the giant thing was turned around, and there were some great spoils, too.
 xQUEST_20150317_000142	Old Potter
-xQUEST_20150317_000143	I say a prayer for my earthware, one by one. {nl}Then one day, the pots will be filled with the goddess' blessings.
-xQUEST_20150317_000144	Swordsmen should have Strength, but also consider Dexterity and Constitution. {nl}You'll need to think about what to start with.
-xQUEST_20150317_000145	Wizards work with Intelligence, but shouldn't forget Spirit for more spellcasting. {nl}You'll have to choose between being balanced or being focused.
-xQUEST_20150317_000146	Archers should have Strength, but Dexterity will prevent problems with movement. {nl}You'll need to think about what to start with.
-xQUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want, first.
+xQUEST_20150317_000143	I put a prayer on my earthware, one by one. {nl}That way, the pots will one day be filled with the goddess' blessings.
+xQUEST_20150317_000144	Sword fighters should have Strength, but you can also have Dexterity and Constitution. {nl}You'll need to think about what to start with.
+xQUEST_20150317_000145	Mage types work with Intelligence, but shouldn't forget Spirit for more spellcasting. {nl}You'll have to choose between being balanced or being focused.
+xQUEST_20150317_000146	Bow users should have Strength, but Dexterity will prevent problems with movement. {nl}You'll need to think about what to start with.
+xQUEST_20150317_000147	Clergy types work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want, first.
```

Comment:
It's referring to the series of classes, not a single class. Since "Swordsman", "Wizard", "Archer", and "Cleric" are game classes, I tagged those words as reserved and tried to find another way to write the line.